Demo case

brunobottazzini opened the pull request.

Commit:
Add demo repository

This patch also changes .gitignore

Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>

Commit:
Library tab comes back to be always shown

Since now there is a demo let's show it from the beggining

Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>

Commit:
Do not allow demo to be saved after edited

Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>

anselmolsm commented:
@brunobottazzini at first, no code should be distributed without the license headers. It seems ace has support for region folding, check if you can fold the license header by default.

brunobottazzini commented:
Good Idea,

I've run some tests here and it is possible.

I will send another version of the patch with only the source codes and with headers folding automatically

Commit:
Fold comentaries for demo case only

Since the license header is too big let's fold it to be
more readable when running the demo case.

Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>
